src/adapter/src/catalog/config.rs

```diff
@@ -132,6 +135,16 @@ impl Default for ClusterReplicaSizeMap {
                     workers: scale.into(),
                 },
             );
+
+            inner.insert(
```

Comment:
Does the comment at the top of this method need to be updated?

Reply:
Yes, thanks, updated.